src/status_im/contexts/settings/wallet/keypairs_and_accounts/remove/view.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.remove.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im.common.standard-authentication.core :as standard-auth]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.remove.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [name key-uid]}]
+  (let [on-remove (rn/use-callback #(rf/dispatch
+                                     [:wallet/remove-keypair key-uid])
+                                   [key-uid])]
+    [:<>
+     [quo/page-top
+      {:container-style style/header-container
+       :title           (i18n/label :t/remove-key-pair-and-derived-accounts)
+       :title-size      :heading-2
+       :description     :context-tag
+       :context-tag     {:type    :icon
+                         :size    24
+                         :context name
+                         :icon    :i/seed-phrase}}]
```

Comment:
@mohsen-ghafouri - We need to use the `drawer-top` component here. That has the `heading-2` text size.

<img width="685"  src="https://github.com/status-im/status-mobile/assets/19339952/a818c777-158c-46bc-9037-96944722b94e">

Reply:
I think `drawer-top` doesn't support icon, i will update it and use it here

Reply:
@smohamedjavid thanks for pointing it out, I replaced